src/lib/pmd/CommandLineSupport.ts

```diff
@@ -0,0 +1,66 @@
+import { Logger } from '@salesforce/core';
+import {AsyncCreatable} from '@salesforce/kit';
+import {ChildProcessWithoutNullStreams} from 'child_process';
+import cspawn = require('cross-spawn');
+
```

Comment:
This class is the parent of CpdWrapper and PmdSupport, it takes the methods out of PmdSupport that we also want to use for Cpd and defines them here.